CalendarApp/NUnitTestProject/BookingServiceTests.DeleteTest.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Linq;
+using BookingRoom.Models.GoogleEvent;
+using Google.Apis.Calendar.v3;
+using NUnit.Core;
+using NUnit.Framework;
+using BookingRoom.Helpers;
+using Microsoft.Practices.Unity;
+using GoogleCalendarService;
+using BookingRoom.Models;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests
+    {
+        private CalendarService _calendarService;
+        private BookingService _meetingBooking;
+        private EventFactory _eventFactory;
+
+        [TestFixtureSetUp]
+        public void TestFixtureSetUp()
+        {
+            var _unityContainer = Dependency.UnityConfig.GetUnityContainer();
+            _calendarService = _unityContainer.Resolve<CalendarService>();
+            _meetingBooking = _unityContainer.Resolve<BookingService>();
+            _eventFactory = _unityContainer.Resolve<EventFactory>();
+        }
+
+        [Test]
+        public void DeleteEvent_WithWrongCalendarId_ThrowsException()
+        {
+            var eventId = "abqq3960anfn5cmi1242usco";
+            var calendarId = string.Empty;//error
+
+            Assert.Throws(Is.InstanceOf<Google.GoogleApiException>().And.Message.EqualTo("Parameter validation failed for \"calendarId\""),
+                () => { _meetingBooking.DeleteEvent(calendarId, eventId); });
+        }
+
+        [Test]
+        public void DeleteEvent_WithWrongEventId_ThrowsException()
+        {
+            var eventId = string.Empty;//error
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            Assert.Throws(Is.InstanceOf<Google.GoogleApiException>().And.Message.EqualTo("Parameter validation failed for \"eventId\""),
```

Comment:
NUnit has Assert.Throws<T> where generic T is type of the exception that is expected